src/main/java/frc/robot/commands/autos/OneFivePlusBump.java

```diff
@@ -0,0 +1,113 @@
+  private static final double DISTANCE_AT_CONE_METERS = 5.9;
+  private static final double NORM_SPEED_INTAKING = 0.3;
+  private double startXMeters = 0;
+  private PathPlannerTrajectory firstTraj =
+      PathPlanner.loadPath(
+          "OnePlusBumpFirstHalf",
+          new PathConstraints(
+              Cal.SwerveSubsystem.VERY_SLOW_LINEAR_SPEED_METERS_PER_SEC,
+              Cal.SwerveSubsystem.VERY_SLOW_LINEAR_ACCELERATION_METERS_PER_SEC_SQ));
+  private PathPlannerTrajectory secondTraj =
+      PathPlanner.loadPath(
+          "OnePlusBumpSecondHalf",
+          new PathConstraints(
+              Cal.SwerveSubsystem.VERY_SLOW_LINEAR_SPEED_METERS_PER_SEC,
+              Cal.SwerveSubsystem.VERY_SLOW_LINEAR_ACCELERATION_METERS_PER_SEC_SQ));
+
+  public OneFivePlusBump(
+      boolean red,
+      boolean fast,
+      Lift lift,
+      Intake intake,
+      DriveSubsystem drive,
+      Lights lights,
+      IntakeLimelight limelight,
+      ScoringLocationUtil scoringLocationUtil) {
+    if (red) {
+      // default to blue, only change for red
+      firstTraj =
+          PathPlanner.loadPath(
+              "OnePlusBumpFirstHalfRed",
+              new PathConstraints(
+                  Cal.SwerveSubsystem.VERY_SLOW_LINEAR_SPEED_METERS_PER_SEC,
+                  Cal.SwerveSubsystem.VERY_SLOW_LINEAR_ACCELERATION_METERS_PER_SEC_SQ));
+
+      secondTraj =
+          PathPlanner.loadPath(
+              "OnePlusBumpSecondHalfRed",
+              new PathConstraints(
+                  Cal.SwerveSubsystem.VERY_SLOW_LINEAR_SPEED_METERS_PER_SEC,
+                  Cal.SwerveSubsystem.VERY_SLOW_LINEAR_ACCELERATION_METERS_PER_SEC_SQ));
+
+      firstTraj =
+          PathPlannerTrajectory.transformTrajectoryForAlliance(
+              firstTraj, DriverStation.Alliance.Red);
+
+      secondTraj =
+          PathPlannerTrajectory.transformTrajectoryForAlliance(
+              secondTraj, DriverStation.Alliance.Red);
+    }
+
+    addRequirements(lift, intake, drive, limelight);
+
+    /** Initialize sequential commands that run for the "15 second autonomous phase" */
+    addCommands(
+        new ScoreThisGamePiece(fast, lift, lights),
+        new InstantCommand(
+            () -> {
+              startXMeters = drive.getPose().getX();
+            }),
+        drive.followTrajectoryCommand(firstTraj, true),
+        // Turn to cone, intake it
+        new InstantCommand(
+            () -> {
+              drive.offsetCurrentHeading(limelight.getAngleToConeDeg());
+            }),
+        new ParallelDeadlineGroup(
+            new SequentialCommandGroup(
+                new WaitCommand(0.5),
+                new RunCommand(
+                        () -> {
+                          drive.rotateOrKeepHeading(NORM_SPEED_INTAKING, 0, 0, false, -1);
+                        })
+                    .until(
+                        () -> (drive.getPose().getX() - startXMeters) > DISTANCE_AT_CONE_METERS)),
```

Comment:
I think we need to drive back after the `ParallelDeadlineGroup`, then run the second half trajectory

Reply:
Also same comment as the other command about using DriveDistance